uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedItem.kt

```diff
@@ -80,6 +82,7 @@ fun FeedItem(
                     bottom.linkTo(parent.bottom, 12.dp)
                 }
                 .width(96.dp)
+                .clip(shape = RoundedCornerShape(4.dp))
 //                .aspectRatio(16F / 9F)
```

Comment:
Should I delete this comment?
It does not seem to represent the current situation.

Reply:
Yes can you delete it?

```suggestion
```

Reply:
Anyway, I wii merge this PR!